codec/enforceproto/testdata/svc.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package service;
+
+option go_package = ".;testdata";
+
+message Customer1 {
+    int32 id = 1;
+    string name = 2;
+    float subscription_fee = 3;
+}
```

Comment:
Let's add cases for:
* nested messages
* message nested via `Any`
* nested messages via `oneof`

Reply:
Thank you @aaronc, done!